backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5575390_sys_gh10475_DontUseWindowFunctionBecauseBadPerf.sql

```diff
@@ -0,0 +1,10 @@
+-- 2020-12-23T13:57:25.249Z
+-- URL zum Konzept
+UPDATE AD_Column SET ColumnSQL='select count(1) from M_ReceiptSchedule rr where rr.c_order_id = M_ReceiptSchedule.c_order_id',Updated=TO_TIMESTAMP('2020-12-23 15:57:25','YYYY-MM-DD HH24:MI:SS'),UpdatedBy=100 WHERE AD_Column_ID=571516
```

Comment:
hm, not sure if we don't *also* need the index on M_ReceiptSchedule.C_Order_ID..

Reply:
we do not